integreat_cms/cms/models/feedback/imprint_page_feedback.py

```diff
@@ -20,7 +20,11 @@ def object_name(self):
         :rtype: str
         """
         try:
-            return self.region.imprint.get_translation(self.language.slug).title
+            if self.language in self.region.languages:
+                return self.region.imprint.get_public_translation(
+                    self.language.slug
+                ).title
+            return self.region.imprint.default_translation.title
```

Comment:
```suggestion
            return self.region.imprint.get_public_translation(
                self.language.slug
            ).title
```
The if-statement does not guarantee that the imprint translation exists in this region, since a region could e.g. just have a german imprint while supporting other languages. However, it looks like just using `get_public_translation` may be enough in this case, because it returns a fallback translation for imprints when there is no translation of the actual language found.

Reply:
```suggestion
            translation = (
                self.region.imprint.get_translation(self.language.slug)
                or self.region.imprint.default_translation
            )
            return translation.title
```
In addition to @david-venhoff's comment: In the case that the imprint only exists in draft state, `get_public_translation()` does not return a translation, even if fallback translations are enabled.

Reply:
Thanks for your feedbacks! I'm sorry for struggling around with such a small issue, I'm getting really confused with the possible ways of translation and their impact in different scenarios. 🙈  I think @timoludwig 's suggestion works for me in that it does not produce an error anymore. 
However I have a few more questions: 
1. I implemented `if self.language in self.region.languages:` because at the moment the API allows to send a feedback in a language, that does not exist for this region (means it does not exist in the regions language tree). Therefore e.g. in @david-venhoff's example if someone would send an API-call with a language, that exists in the database in general, but not for the specific region, an error would show up again. With Timo's soultion this would not happen, but doesn't it still make sense to not allow this in the API and return a 404 instead? 
2. A more general question: What is the proper meaning of a fallback translation in the Integreat context? Is it always the translation of the default language of a region or is it the next parent translation of a language in the language tree, that exists? E.g say that 
```
DE (default)
   EN 
      ES 
```
is the language tree and a page exists in DE and EN, what is the fallback translation for ES? 
I'm sorry, I just found that I really have a lack of understanding here. 👀

Reply:
> I'm sorry for struggling around with such a small issue, I'm getting really confused with the possible ways of translation and their impact in different scenarios.

No worries, we should probably write more documentation on this topic! 

> 1. But doesn't it still make sense to not allow this in the API and return a 404 instead? 

Yes, correct, in the API this makes sense, but for the representation of the existing objects I wouldn't limit it, since e.g. a feedback entry could be created while a language is active, then the language is disabled but the feedback entry still exists in the database and somehow needs to be displayed in the list.

> 2. What is the proper meaning of a fallback translation in the Integreat context? Is it always the translation of the default language of a region? 

Yes, always the default language! The order of the tree only counts for the internal translation management and has no influence on the content delivered to the users.

Reply:
Okay, thank you very much for clarification, then I will just add the additional catch in the API.